brunoais/apispec/ext/decorated_crawler.py

```diff
@@ -1,219 +1,26 @@
 # -*- coding: utf-8 -*-
-"""Operations helper that allows you to pass a decorated view and get the combined documentation of all decorator functions.
-It required the view passed to `add_path`. Inspects view docstrings and its docd_wraps decorator functions and merges all the
-documentation into a single document.
+"""
+Operations helper that allows you to pass a decorated view and get the combined documentation of all decorator
+functions.
```

Comment:
To make a markdown line change, you need to add two spaces to the end of the string. Did you only mean a new line in the file or did you also meant to have it when reading?

Reply:
A lot of the extra line breaks I've added are just to make flake8 happy with a 120 character max width - bearing in mind your easy-to-read point below, do you think this makes things clearer or more confusing?

Reply:
Looks good for me. I just don't know if it is also meant to have the new line in the final script or not.